### ISSUE TITLE
Docs: app/error.tsx could not catch errors in the route groups.

### ISSUE THREAD

**Issue by jagvaandanzan** [vercel/next.js#52113]
### What is the improvement or update you wish to see?

In app routes - error handling section it says - "Errors bubble up to the nearest parent error boundary. This means an error.js file will handle errors for all its nested child segments". But it does not work with route groups. If I am using many route groups, I need to create error.tsx file that much. If it is right way to do it, would you include it in the docs?

### Is there any context that might help us understand?

My project's structure:
- app/
- - error.tsx // could not catch
- - dashboard/
- - - (admin)/
- - - - error.tsx // catches error
- - - - panel/
- - - - - page.tsx // throws error

### Does the docs page already exist? Please link to it.

https://nextjs.org/docs/app/building-your-application/routing/error-handling

<sub>[DX-1727](https://linear.app/vercel/issue/DX-1727/docs-apperrortsx-could-not-catch-errors-in-the-route-groups)</sub>

**Comment by balazsorban44** [comment]
Good point, I think we could clarify the error handling for route groups as well.

**Comment by jagvaandanzan** [comment]
btw loading ui works same way

**Comment by Zheng-Jie-Gan** [comment]
Any updates on this ?

**Comment by marcoshernanz** [comment]
Thanks for the report. I couldn't reproduce this using the structure shown, including on Next 13.4.7 and current canary. The root `app/error.tsx` handled the page error on direct loads and client navigation, and root `loading.tsx` also applied through the route group. The docs now describe the nesting and same-segment layout exception, so I'm closing this as not reproducible. If this still occurs, please open a new issue with a minimal repo and environment details.
